lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -277,7 +273,8 @@ func (handler *ContainersHandlersImpl) GetContainerListHandler(params containers
 		info.ContainerID = &container.ExecConfig.ID
 		info.LayerID = &container.ExecConfig.LayerID
 		info.Created = &container.ExecConfig.Created
-		info.Status = &container.Status
+		friendly := container.FriendlyStatus()
```

Comment:
How about `conatiner.Status.String()`, i.e. implement the [Stringer](https://github.com/vmware/vic/issues/1701) interface for `Status`?